model/config.go

```diff
@@ -213,6 +213,9 @@ type ServiceSettings struct {
 	CloseUnusedDirectMessages                *bool
 	EnablePreviewFeatures                    *bool
 	EnableTutorial                           *bool
+	ImageProxyType                           *string
+	ImageProxyURL                            *string
+	ImageProxyOptions                        *string
```

Comment:
You might consider adding a check to the `isValid()` method to verify that a valid proxy type has been selected. I'd hate to be configuring this, make a spelling mistake, and have the server start up with no complaints, but my proxy not work as desired.

Reply:
+1. I'll be sure to add that.